lldb/test/API/functionalities/statusline/main.c

```diff
@@ -0,0 +1,11 @@
+int bar(int b) { return b * b; }
+
+int foo(int f) {
+  int b = bar(f); // Break here
+  return b;
+}
+
+int main() {
+  int f = foo(42);
+  return f;
+}
```

Comment:
Does this need many functions? I suppose having at least one might avoid some systems where the name of main is mangled somehow, but is there another reason?